assets/components.css

```diff
@@ -2,6 +2,12 @@
 @import url("./index.css");
 @import url("./documentation.css");
 
+
```

Comment:
suggestion: remove blank spaces if you have time.

Reply:
It is so that there is no conflicting lines while merging other branches.